velox/connectors/hive/tests/PartitionIdGeneratorTest.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/connectors/hive/PartitionIdGenerator.h"
+#include "gtest/gtest.h"
+#include "velox/vector/tests/utils/VectorTestBase.h"
+
+namespace facebook::velox::connector::hive {
+
+class PartitionIdGeneratorTest : public ::testing::Test,
+                                 public test::VectorTestBase {
+ public:
+  void SetUp() override {
+    pool_ = memory::getDefaultMemoryPool();
```

Comment:
This is not needed since VectorTestBase provides a pool.

Reply:
Good to know.